src/pmi/src/pmix.c

```diff
@@ -422,6 +422,10 @@ static const char *attribute_from_key(const char *key)
 {
     if (!strcmp(key, "PMI_hwloc_xmlfile")) {
         return key;
+    } else if (!strcmp(key, "PMI_mpi_memory_alloc_kinds")) {
+        return key;
+    } else if (!strcmp(key, PMIX_FABRIC_COORDINATES)) {
+        return key;
```

Comment:
Getting unsupported pmi key should not automatically abort the application. I think the fix should be in mpich where it is querying the key.

Reply:
Support is determined by the PMI server, no? How does MPICH know if a key is supported?

Reply:
If you get a value, it means it is supported. Otherwise, it is not. Am I missing something?

Reply:
Changed based on offline discussion.